### DISCUSSION TITLE
How to update notifications silently in Android?

### DISCUSSION THREAD

**Discussion by thefrol** [thudugala/Plugin.LocalNotification#206] [Q&A]
Hello!

I am looking for a way to update my ongoing notification without notification sound in Andoid

ATM i have one notification id and one channel. And i want to update my notification maybe every 10 seconds, but it always fires the sound. It there is a way to make a new notification without too much sound going on?

**Comment by thudugala** [comment]
@thefrol Have try setting Importance to NotificationImportance.Low when creating the channel

Also remember to delete already created channel
